src/SDK.ts

```diff
@@ -137,9 +137,15 @@ export default class SDK implements ISDK {
     });
     const elapsedTimeInMilliseconds = timer.milliSecondsElapsed;
     const { data } = response;
+
+    if (data.LiveChatVersion && data.LiveChatVersion === LiveChatVersion.V3) {
+      this.liveChatVersion = data.LiveChatVersion;
+    }
+
     if (data.LiveChatVersion && data.LiveChatVersion === LiveChatVersion.V2) {
       this.liveChatVersion = data.LiveChatVersion;
     }
```

Comment:
couldnt this be simplified with an OR?

Reply:
I mean for V2 and V3 -> 

  if (data.LiveChatVersion && (data.LiveChatVersion === LiveChatVersion.V3 || data.LiveChatVersion === LiveChatVersion.V2) ) {
      this.liveChatVersion = data.LiveChatVersion;
    }

Reply:
Refactored to simply assign the version regardless of the version. It would work with any versions in the future